packages/web-components/src/global/color-mode.css

```diff
@@ -605,6 +605,28 @@ ic-theme,
   --ic-skeleton-background: var(--ic-architectural-100);
   --ic-skeleton-background-secondary: var(--ic-architectural-200);
 
+  /* Side navigation  */
```

Comment:
Since it's the same tokens for light and dark, these tokens can be moved into colors.css like we've done for classification banner, hero, toast etc.